bitcoin/test/fuzz/miniscript_random.cpp

```diff
@@ -887,3 +924,16 @@ FUZZ_TARGET_INIT(miniscript_string, FuzzInit)
     assert(parsed2);
     assert(*parsed == *parsed2);
 }
+
+/* Fuzz tests that test parsing from a script, and roundtripping via script. */
+FUZZ_TARGET_INIT(miniscript_script, FuzzInit)
```

Comment:
nit: I don't think that the `miniscript_script` target needs `FuzzInit` since it doesn't use anything from TEST_DATA.

Reply:
You're right, fixed.